docker/model-server/Dockerfile

```diff
@@ -7,6 +7,9 @@ RUN apt-get update --fix-missing && \
 ARG BENTOML_VERSION
 ENV BENTOML_VERSION=$BENTOML_VERSION
 
+# Improve conda pip interoperability
+RUN conda config --set pip_interop_enabled True
```

Comment:
What about custom base image that has conda?

Reply:
Probably a better place is the Bento-init file